cmd/empire/main.go

```diff
@@ -274,11 +270,29 @@ var EmpireFlags = []cli.Flag{
 		EnvVar: "EMPIRE_LOGS_STREAMER",
 	},
 	cli.StringFlag{
+		Name:   FlagEventsBackend,
+		Value:  "",
+		Usage:  "The backend implementation to use to send event notifactions",
+		EnvVar: "EMPIRE_EVENTS_BACKEND",
+	},
+	cli.StringFlag{
+		Name:   FlagRunLogsBackend,
+		Value:  "",
+		Usage:  "The backend implementation to use to record the logs from interactive runs",
+		EnvVar: "EMPIRE_RUN_LOGS_BACKEND",
+	},
+	cli.StringFlag{
 		Name:   FlagSNSTopic,
 		Value:  "",
 		Usage:  "When using the SNS events backend, this is the SNS topic that gets published to",
 		EnvVar: "EMPIRE_SNS_TOPIC",
 	},
+	cli.StringFlag{
+		Name:   FlagCloudWatchLogGroup,
+		Value:  "",
+		Usage:  "When using the CloudWatch backend for run logs, this is the log group that CloudWatch log streams will be created in.",
```

Comment:
I'd specify the command line arg.  `When specifying the 'cloudwatch' backend for FlagRunLogsBackend...`